app/jobs/search/index_job.rb

```diff
@@ -0,0 +1,14 @@
+module Search
+  class IndexJob < ApplicationJob
+    queue_as :search_index
+
+    def perform(record_type, record_id)
+      return unless %w[Comment Article User].include?(record_type)
```

Comment:
I think we should raise an error here if the `record_type` is not found since that indicates either we need to have it included here or something it trying to serialize something it shouldnt. I could see myself trying to use this for PageViews and forgetting to add that model here.

Reply:
Cool, I thought of this initially but haven't implemented.